src/Workspaces/Core/Portable/Indentation/IInferredIndentationService.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Host;
+using Microsoft.CodeAnalysis.Options;
+
+namespace Microsoft.CodeAnalysis.Indentation
+{
+    internal interface IInferredIndentationService : IWorkspaceService
```

Comment:
Can we add an XML doc comment here just to explain this? Doesn't have to be much, but somebody not familiar with this might need a bit of a hint.